src/components/Section/ProposalVoting/DelegationsLabel.tsx

```diff
@@ -2,25 +2,69 @@ import React from 'react'
 
 import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
 import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Popup } from 'decentraland-ui/dist/components/Popup/Popup'
+
+import Info from '../../Icon/Info'
+import VotingPower from '../../Token/VotingPower'
+import Username from '../../User/Username'
 
 import './DelegationsLabel.css'
 
 export interface DelegationsLabelProps {
   delegateLabel?: { id: string; values?: any }
   delegatorsLabel?: { id: string; values?: any }
+  infoMessage?: { id: string; values?: any }
```

Comment:
Can't this be typed? `{ delegate; own_vp; delegators_that_voted; total_delegators; delegators_without_vote }` considering which ones are optional